README.md

```diff
@@ -104,99 +106,53 @@ npm run build && node ./build/src/index.js
 
 ### Deploy your smart contract
 
 ```sh
 zk deploy <alias>
 // OR
-zk deploy // will show a list of aliases in your project to choose from
+zk deploy // shows a list of aliases in your project to choose from
 ```
 
-_**Deployment is possible to Berkeley Testnet currently. It is not possible to deploy
-to Mina Mainnet at this time.**_
+_**Deployment is supported only to Berkeley Testnet. 
+zkApp programmability is not yet available on the Mina Mainnet.**_
 
 The `deploy` command allows you to deploy a smart contract to your desired
-alias. You must run `zk config` once before deploying, in order to set up a
+alias. You must run `zk config` once before deploying to set up a
 deploy alias with the required details. Then run `zk deploy <alias>` or `zk deploy` (and select the alias from the list shown) and type `yes` or `y` to
 confirm when prompted.
 
-Note: When deploying to an alias for the first time, the CLI will prompt you to
-choose which smart contract you want to deploy from those that exist as _named_
-exports in your project. The name of the smart contract that you choose will
-then be remembered by being saved into your `config.json` for this alias, so
-that running `zk deploy <alias>` will automatically deploy this _same_ smart
-contract in the future when deploying to this alias, for safety.
-
-## Contributing
-
-The best way to contribute to the zkApp CLI is to help us test it broadly to
-ensure it works as expected on all platforms. If you encounter any issues,
-please submit an issue on Github and include the info printed when running `zk system`, which contains your OS, NodeJS, & zkapp-cli versions to help us
-reproduce the issue.
-
-To submit a PR:
-
-```sh
-# Visit https://github.com/o1-labs/zkapp-cli & fork it.
-git clone https://github.com/<your-username>/zkapp-cli.git
-cd zkapp-cli
-git remote add upstream https://github.com/o1-labs/zkapp-cli.git
-npm install
-npm link # makes it available globally on your system
-
-# You can also use the following commands if you have previously run `npm link`
-# against the different version of `zkapp-cli` (say, after the branches switch):
-# npm r zkapp-cli -g && npm install && npm run build --if-present && npm link
-
-git checkout -b upstream/main
-# Make desired changes and commit
-git push origin <your-branch>
-# Submit a pull request
-# To switch back to the released version, run `npm i -g zkapp-cli`
-```
+Note: When you deploy to an alias for the first time, you are prompted to choose which smart contract you want to deploy from those that exist as _named_ exports in your project. The name of the smart contract that you choose is remembered by being saved into your `config.json` for this alias. For safety, the next time you run `zk deploy <alias>` it automatically deploys this _same_ smart contract in the future when deploying to this alias.
 
 ## License
 
-[Apache-2.0](LICENSE)
+[Apache-2.0](https://github.com/o1-labs/zkapp-cli/blob/main/LICENSE)
```

Comment:
the README is scraped to create https://www.npmjs.com/package/zkapp-cli 

I learned that README links must be absolute to work on npm